components/overflow-group/overflow-group.js

```diff
@@ -81,7 +90,7 @@ function createMenuItemMenu(node) {
 	</d2l-menu-item>`;
 }
 
-function convertToDropdownItem(node) {
+function convertToOverflowItem(node) {
```

Comment:
Given a slightly more generic name since this was a function that would then be created by the components using the mixin